README-mux.md

```diff
@@ -0,0 +1,345 @@
+    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-origin-15-aio > ansible-inventory
+
+To use ViaQ on Red Hat OCP, use the
+[ansible-inventory-ocp-35-aio](ansible-inventory-ocp-35-aio) file:
+
+    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-ocp-35-aio > ansible-inventory
+
+Copy `vars.yaml.template` to `vars.yaml`.  You will need to change the
+following fields in `vars.yaml`:
+
+* `ansible_ssh_user` - this is the user created in
+  [provisioning](#provisioning-a-machine-to-run-viaq) which can use
+  passwordless ssh
+* `ansible_become` - use `no` if `ansible_ssh_user` is `root`, otherwise,
+  use `yes`
+* `openshift_master_default_subdomain` - this is the public subdomain to use
+  for all of the external facing logging services, such as the OpenShift UI,
+  Kibana, mux, and Elasticsearch.  By default, the OpenShift UI will be
+  accessed via `https://openshift.{{ openshift_master_default_subdomain }}:8443`,
+  Kibana will be at `https://kibana.{{ openshift_master_default_subdomain }}`,
+  etc.
+* `openshift_public_ip` - this is the public IP address, the IP address used in
+  your internal DNS or host look up for browsers and other external client
+  programs.  For example, in OpenStack, this will be the *floating ip* address
+  of the machine.  This may be the same as the `eth0` IP addrss of the machine,
+  in which case, just use `"{{ ansible_eth0.ipv4.address }}"` as the value as
+  is done for `openshift_ip`
+* `openshift_hostname` - this is the private hostname of the machine that will
+  be used inside the cluster.  For example, OpenStack machines will have a
+  "private" hostname assigned by Neutron networking.  This may be the same as
+  the external hostname if you do not have a "private" hostname - in that case,
+  just use `openshift.{{ openshift_master_default_subdomain }}`
+
+You can also override variables in the inventory by setting them in `vars.yaml`.
+
+The public hostname would typically be a DNS entry for the
+public IP address, but you can "fake" these out with `/etc/hosts` entries:
+
+    10.16.19.171 openshift.logging.test kibana.logging.test mux.logging.test
+
+Where `10.16.19.171` is the external/public IP address of the machine you created in
+[provisioning](#provisioning-a-machine-to-run-viaq), and the hostnames are
+various aliases you created to access the machine externally via Ansible,
+Kibana, and mux.  That is, unless you have configured DNS or some other host
+lookup service, you will need to make similar changes to `/etc/hosts` from all
+client machines from which you will use Kibana, or access the Elasticsearch API
+directly, or from which you will send logs to the mux.
+
+Once you have your inventory and `vars.yaml`, you can run ansible:
+
+    # cd /usr/share/ansible/openshift-ansible
+    # (or wherever you cloned the git repo if using git)
+    # ANSIBLE_LOG_PATH=/tmp/ansible.log ansible-playbook -vvv \
+      -e @/path/to/vars.yaml \
```

Comment:
If you direct the user where to save the files you can set here the same path.
If not, state this should be updated.